dotnet/src/SemanticKernel.Core/KernelBuilder.cs

```diff
@@ -46,7 +46,7 @@ public Kernel Build()
         );
 #pragma warning restore CS8604 // Possible null reference argument.
 
-        instance.SetCulture(this._culture ?? instance.GetCulture());
+        instance.Culture = this._culture ?? instance.Culture;
```

Comment:
Is the `?? instance.Culture` part necessary?

Reply:
nope, not really, just wanted to avoid if operator and not making Kernel.Culture property nullable.

Reply:
addressed with if operator